src/components/buttons/SecondaryBtn.tsx

```diff
@@ -13,14 +13,16 @@ export const SecondaryBtn = ({
   outlined,
   padding,
   variant,
+  height,
 }: {
   startIcon?: ReactNode
   buttonContent: ReactNode
-  handleClick?: (() => void) | ((e: React.MouseEvent) => void)
+  handleClick?: (() => void) | ((e: React.MouseEvent) => void) | ((e: React.MouseEvent<HTMLButtonElement>) => void)
```

Comment:
As this is a button component (HTMLButtonElement), why still use the non generic version?